translator/GCP_cloud_function/cloud_function/icone_translator.py

```diff
@@ -1,11 +1,12 @@
+import xmltodict
```

Comment:
This seems to be a previous version without your latest changes. Did you pull before creating this PR?